src/__test__/containers/SignUpContainer.test.jsx

```diff
@@ -0,0 +1,27 @@
+import configureStore from 'redux-mock-store';
+import React from 'react';
+import { mount } from 'enzyme';
+import { Button } from 'react-materialize';
+import SignUpContainer from '../../containers/Auth/SignUpContainer';
+
+
+const mock = configureStore();
+const store = mock({
+  signUp: {
+    loading: false,
+    success: false,
+  },
+});
+
+describe('test signup container', () => {
+  test('test container mounts', () => {
```

Comment:
I thought we agreed on using the describe-it block for tests... Kindly change this for consistency

Reply:
Done